3515/2/helper_modules/photo_download_uri_method.rb

```diff
@@ -0,0 +1,5 @@
+module PhotoDownloadUri
```

Comment:
I guess it's too much have 1 method in one module that have the same name. It's better to move all such methods in `Helper` module